src/main/java/fr/openent/minibadge/controller/FakeRight.java

```diff
@@ -39,4 +39,10 @@ public void statisticsView(HttpServerRequest request) {
     public void statisticsViewAllStructures(HttpServerRequest request) {
         notImplemented(request);
     }
+
+    @Get("/rights/admin")
+    @SecuredAction(Rights.ADMIN)
+    public void admin(HttpServerRequest request) {
+        notImplemented(request);
+    }
```

Comment:
j'suis très moyen fan mais comme je vois que le reste du fichier a été fait pareil je comprends

J'aurais préféré un truc du genre sinon :

```suggestion

    @SecuredAction(Rights.ADMIN)
    public void admin(HttpServerRequest request) {}
```

comme ça y'a même pas de route appelable, c'est littéralement une fonction d'init et pas une vraie route qui répond notImplemented